### PR TITLE
Polish

### PR DESCRIPTION
**Pull request by izeye**
This PR fixes some typos and polishes trivial stuff.

### CODE REVIEW

File: spring-boot-project/spring-boot-actuator-autoconfigure/src/main/java/org/springframework/boot/actuate/autoconfigure/metrics/MetricsProperties.java [hunk 1]
```diff
@@ -42,7 +41,7 @@
 	 * Whether meter IDs starting-with the specified name should be enabled. The longest
 	 * match wins, the key `all` can also be used to configure all meters.
 	 */
-	private Map<String, Boolean> enable = new LinkedHashMap<>();
+	private final Map<String, Boolean> enable = new LinkedHashMap<>();
```
Comment (snicoll) [inline spring-boot-project/spring-boot-actuator-autoconfigure/src/main/java/org/springframework/boot/actuate/autoconfigure/metrics/MetricsProperties.java]: Can you please move the removal of Map setters to a separate PRs? I'd like to merge the rest in `2.0.x` but I can't merge this there as it's an API breaking change.

File: spring-boot-project/spring-boot-actuator-autoconfigure/src/main/java/org/springframework/boot/actuate/autoconfigure/metrics/MetricsProperties.java [hunk 2]
```diff
@@ -188,35 +182,20 @@ public void setRequestsMetricName(String requestsMetricName) {
...
 		 * specified).
 		 */
-		private Map<String, ServiceLevelAgreementBoundary[]> sla = new LinkedHashMap<>();
+		private final Map<String, ServiceLevelAgreementBoundary[]> sla = new LinkedHashMap<>();
 
 		public Map<String, Boolean> getPercentilesHistogram() {
 			return this.percentilesHistogram;
 		}
 
-		public void setPercentilesHistogram(Map<String, Boolean> percentilesHistogram) {
```
Comment (snicoll) [inline spring-boot-project/spring-boot-actuator-autoconfigure/src/main/java/org/springframework/boot/actuate/autoconfigure/metrics/MetricsProperties.java]: this too

File: spring-boot-project/spring-boot-actuator-autoconfigure/src/main/java/org/springframework/boot/actuate/autoconfigure/metrics/MetricsProperties.java [hunk 3]
```diff
@@ -188,35 +182,20 @@ public void setRequestsMetricName(String requestsMetricName) {
...
-			Assert.notNull(percentilesHistogram, "PercentilesHistogram must not be null");
-			this.percentilesHistogram = percentilesHistogram;
-		}
-
 		public Map<String, double[]> getPercentiles() {
 			return this.percentiles;
 		}
 
-		public void setPercentiles(Map<String, double[]> percentiles) {
```
Comment (snicoll) [inline spring-boot-project/spring-boot-actuator-autoconfigure/src/main/java/org/springframework/boot/actuate/autoconfigure/metrics/MetricsProperties.java]: And this

File: spring-boot-project/spring-boot-actuator-autoconfigure/src/main/java/org/springframework/boot/actuate/autoconfigure/metrics/MetricsProperties.java [hunk 4]
```diff
@@ -188,35 +182,20 @@ public void setRequestsMetricName(String requestsMetricName) {
...
-			Assert.notNull(percentiles, "Percentiles must not be null");
-			this.percentiles = percentiles;
-		}
-
 		public Map<String, ServiceLevelAgreementBoundary[]> getSla() {
 			return this.sla;
 		}
 
-		public void setSla(Map<String, ServiceLevelAgreementBoundary[]> sla) {
```
Comment (snicoll) [inline spring-boot-project/spring-boot-actuator-autoconfigure/src/main/java/org/springframework/boot/actuate/autoconfigure/metrics/MetricsProperties.java]: And this

File: spring-boot-project/spring-boot-docs/src/main/asciidoc/getting-started.adoc [hunk 5]
```diff
@@ -662,7 +662,7 @@ that you have added. Since `spring-boot-starter-web` added Tomcat and Spring MVC
 auto-configuration assumes that you are developing a web application and sets up Spring
 accordingly.
 
-.Starters and Auto-Configuration
```
Comment (snicoll) [inline spring-boot-project/spring-boot-docs/src/main/asciidoc/getting-started.adoc]: I am not sure that we should change that. Titles in the doc are consistently capitalized. Thoughts?

Comment (izeye) [reply spring-boot-project/spring-boot-docs/src/main/asciidoc/getting-started.adoc]: @snicoll Actually I tried to retain the capitalisation there as intended. AFAIK English capitalisation is only for the first letter but I might be wrong as I'm not a native English speaker 😅

Comment (snicoll) [reply spring-boot-project/spring-boot-docs/src/main/asciidoc/getting-started.adoc]: You are correct. This is also consistent with [this section in the doc](https://docs.spring.io/spring-boot/docs/current/reference/htmlsingle/#using-boot-auto-configuration). Thanks for the feedback.

**Comment by izeye** [comment]
@snicoll Thanks for the quick feedback!

I created another PR for the `MetricsProperties` changes: https://github.com/spring-projects/spring-boot/pull/13324

**Pull request closed without merging**